docs/tutorials/solidity/solidity_7_updating_solidity_contracts.md

```diff
@@ -7,7 +7,7 @@ title: "Tutorials | Solidity 7: Updating Solidity Contracts"
 
 # Solidity Series
 
-This sequence of tutorials assumes that you have an understanding of the `eris` tooling to the point we ended in our [101 tutorial sequence](/docs/tutorials/getting-started/).
```

Comment:
so this seems weird to me because the URL structure on site will be a bit different than directory structure in repo